sapi/fpm/tests/bug77185-reload-robustness.phpt

```diff
@@ -1,46 +0,0 @@
---TEST--
```

Comment:
Hm, this test shouldn't be removed, should it?

Reply:
@cmb69 ignore this (commit from rebase, already exists in PHP-7.4, will be ignore in merge)